kp_test.go

```diff
@@ -1293,3 +1292,35 @@ func TestDeleteKey_ForceOptTrue_URLHasForce(t *testing.T) {
 
 	assert.True(t, gock.IsDone(), "Expected HTTP requests not called!")
 }
+
+func TestSetKey_ForDeletion(t *testing.T) {
+        defer gock.Off()
+
+        gock.New("http://example.com").
+                Reply(204)
```

Comment:
```suggestion
               MatchParam("action","setKeyDeletion)).
               Reply(204)
```
it would be good if we can add this to the url to specify the correct url path for set key for deletion.
Same for the unset key for deletion.
The rest looks good.